iwyu_include_picker.cc

```diff
@@ -1378,18 +1378,19 @@ vector<string> IncludePicker::GetCandidateHeadersForSymbolUsedFrom(
 vector<MappedInclude> IncludePicker::GetCandidateHeadersForFilepath(
     const string& filepath, const string& including_filepath) const {
   CHECK_(has_called_finalize_added_include_lines_ && "Must finalize includes");
+  string absolute_quoted_header = ConvertToQuotedInclude(filepath);
```

Comment:
In what sense is this absolute?

Reply:
Oh, is `filepath` always absolute, and `ConvertToQuotedInclude` maintains the full path?

Reply:
It's absolute in the sense that it's not relative to `including_filepath`; it's the "universal" version of this quoted include, which would work for any file in the project.  It won't usually be an absolute path in the usual sense.  I didn't like this choice of name really, but I couldn't think of a better one.  Maybe `universal_quoted_header`?